CHANGELOG.md

```diff
@@ -28,6 +31,7 @@ The main features of this release are:
 - #129 (ark-ff) Move `ark_ff::{UniformRand, test_rng}` to `ark_std::{UniformRand, test_rng}`.
     Importing these from `ark-ff` is still possible, but is deprecated and will be removed in the following release.
 - #144 (ark-poly) Add `CanonicalSerialize` and `CanonicalDeserialize` trait bounds for `Polynomial`.
+- #140 (ark-poly) Move methods `from_coefficients_slice`, `from_coefficients_vec`, `terms` from `MVPolynomial` to `MVPolynomialCoefficientForm`
```

Comment:
Can you move this one entry up? (To make the numbers in sorted order)

Reply:
resolved